lib/_tls_wrap.js

```diff
@@ -971,14 +979,16 @@ Server.prototype.addContext = function(servername, context) {
 };
 
 function SNICallback(servername, callback) {
+  const contexts = this.server._contexts;
   var ctx;
 
-  this.server._contexts.some(function(elem) {
+  for (var i = 0; i < contexts.length; i++) {
+    const elem = contexts[i];
     if (elem[0].test(servername)) {
       ctx = elem[1];
```

Comment:
No need for the ctx variable at all. You could call the callback here and return and call the callback below with undefined (if that is possible - otherwise that is obsolete).